src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -31,22 +65,15 @@ export default class extends Controller {
 
         const isOpen = header.getAttribute('aria-expanded') === 'true';
 
-        // Reset all accordions
-        headerTargets.forEach((header) => {
-            header.setAttribute('aria-expanded', 'false');
-        });
-        arrowTargets.forEach((arrow) => {
-            arrow.classList.remove(...arrowRotateClasses);
-        });
-        contentTargets.forEach((content) => {
-            content.classList.add(...contentHiddenClasses);
-        });
-
         // Open the current accordion if it wasn't open before (else we toggle it)
         if (!isOpen) {
             arrow.classList.add(...arrowRotateClasses);
             header.setAttribute('aria-expanded', 'true');
             content.classList.remove(...contentHiddenClasses);
+        } else {
+            arrow.classList.remove(...arrowRotateClasses);
+            header.setAttribute('aria-expanded', 'false');
+            content.classList.add(...contentHiddenClasses);
```

Comment:
Out of scope: I wonder if the classes are necessary, assuming the HTML Markup here can't be changed and therefore `.myCSSClass[aria-expanded="true"]` and `.myCSSClass:not([aria-expanded="true"])` would be enough to select these states in CSS.

Reply:
@renestalder You mean the arrowRotateClasses and contentHiddenClasses? Those are the tailwind classes that turn the arrow and hide the content when it's collapsed. 
But this could of course also be solved via the CSS file, as you proposed. (Especially since we need the `aria-expanded` anyways)

Reply:
@Ruesa18 Right, I forgot about Tailwind being the party pooper for proper CSS.